eternalcore-core/src/main/java/com/eternalcode/core/feature/teleportrandomplayer/TeleportToRandomPlayerCommand.java

```diff
@@ -0,0 +1,52 @@
+package com.eternalcode.core.feature.teleportrandomplayer;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+
+import org.bukkit.entity.Player;
+
+@Command(name = "teleportorandomplayer", aliases = { "tprp" })
```

Comment:
ja bym tutaj wywalił teleport**to**randomplayer i zostawił samo `tprp`, ewentualnie `tprandom`